tests/foreman/cli/test_docker.py

```diff
@@ -1280,6 +1280,7 @@ def tearDownClass(cls):
         cls.docker_host.destroy()
 
     @run_only_on('sat')
+    @upgrade
```

Comment:
We already have docker client scenario https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_docker.py#L1348. Not sure we should cover this specially for 'docker pull'

Reply:
you are right, i simply missed that